src/Plugin/dgi_migrate_alter/spreadsheet/DssiNodeAlter.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+namespace Drupal\bcelndora\Plugin\dgi_migrate_alter\spreadsheet;
+
+use Drupal\Component\Utility\NestedArray;
+use Drupal\dgi_migrate_alter\Plugin\MigrationAlterBase;
+use Drupal\dgi_migrate_alter\Plugin\MigrationAlterInterface;
+
+/**
+ * Alter for dssi_node migration.
+ *
+ * @MigrationAlter(
+ *   id = "bcelndora_dssi_node_alter",
+ *   label = @Translation("DSSI Node Migration Alteration"),
+ *   description = @Translation("Alters the DSSI Node migration."),
+ *   migration_id = "dssi_node"
+ * )
+ */
+class DssiNodeAlter extends MigrationAlterBase implements MigrationAlterInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function alter(array &$migration) {
+    $process =& $migration['process'];
+
+    $process['field_resource_type'][1]['bundle'] = 'library_of_congress_resource_typ';
+
+    // Add Text (plain) delimited fields.
+    $fields_to_add = [
+      ['field_local_contexts', 'local_contexts', ';'],
+      ['field_access_id', 'access_id', ';'],
+    ];
+
+    foreach ($fields_to_add as $field) {
+      $process[$field[0]] = [
+        [
+          'plugin' => 'skip_on_empty',
+          'source' => $field[1],
+          'method' => 'process',
+        ],
+        [
+          'plugin' => 'dgi_migrate.process.explode',
+          'delimiter' => $field[2],
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => 'Empty ' . $field[1] . '.',
+        ],
+      ];
+    }
+
+    // Add Text (plain) non-delimited fields.
+    $fields_to_add = [
```

Comment:
Not sure where to comment on exactly, but I'm thinking we should generally avoid the variable reuse across these bigger functions.

More specific names, with more specific structures should help future readability.